generators/server/templates/src/test/java/package/config/_WebConfigurerTest.java

```diff
@@ -159,7 +159,7 @@ public void testCustomizeServletContainer() {
         assertThat(container.getMimeMappings().get("json")).isEqualTo("text/html;charset=utf-8");
         <%_ if (!skipClient) { _%>
         if (container.getDocumentRoot() != null) {
-            assertThat(container.getDocumentRoot().getPath()).isEqualTo("target/www");
+            assertThat(container.getDocumentRoot().getPath()).isEqualTo("<%= BUILD_DIR %>www");
```

Comment:
Related to https://github.com/jhipster/generator-jhipster/issues/5805, I'm not sure it will work on Windows